drivers/bluetooth/hci/ipc.c

```diff
@@ -265,19 +266,34 @@ static int bt_ipc_send(const struct device *dev, struct net_buf *buf)
 		break;
 	default:
 		LOG_ERR("Unknown type %u", bt_buf_get_type(buf));
+		err = -ENOMSG;
 		goto done;
 	}
 	net_buf_push_u8(buf, pkt_indicator);
 
 	LOG_HEXDUMP_DBG(buf->data, buf->len, "Final HCI buffer:");
-	err = ipc_service_send(&data->hci_ept, buf->data, buf->len);
+
+	do {
+		err = ipc_service_send(&data->hci_ept, buf->data, buf->len);
+		if (err < 0) {
+			retries++;
+#if (USEC_PER_SEC / CONFIG_SYS_CLOCK_TICKS_PER_SEC) > CONFIG_BT_HCI_IPC_SEND_RETRY_DELAY_US
```

Comment:
`CONFIG_SYS_CLOCK_TICKS_PER_SEC` may be `0`:

```
A value of 0 completely disables timer support in the kernel.
```
Need to guard against a 0-division

Reply:
Added the condition